zebrad/tests/common/regtest.rs

```diff
@@ -73,18 +74,28 @@ async fn submit_blocks(rpc_address: SocketAddr) -> Result<()> {
             NetworkUpgrade::Nu5
         };
 
-        let block_data = hex::encode(
-            proposal_block_from_template(&block_template, TimeSource::default(), network_upgrade)?
-                .zcash_serialize_to_vec()?,
-        );
+        let mut block =
+            proposal_block_from_template(&block_template, TimeSource::default(), network_upgrade)?;
+        let height = block
+            .coinbase_height()
+            .expect("should have a coinbase height");
+
+        while !network.disable_pow()
+            && zebra_consensus::difficulty_is_valid(&block.header, &network, &height, &block.hash())
+                .is_err()
```

Comment:
This isn't strictly necessary yet, but if we leave it here now, we won't have to change it when re-enabling PoW on Regtest later.